### PR TITLE
Run autoencoder experiments

### PR DESCRIPTION
**Pull request by scott-sievert-3**
This PR runs some experiments with autoencoders.

TODO:

- [x] get base model running
  - [x] autoencoder base working
  - [x] tune dataset/model/image IO
- [x] tune hyperparameters
- [ ] generate graphs

### CODE REVIEW

**Comment by scott-sievert-3** [comment]
Here's some example input/output pairs on the validation set:

Screenshot 2024-08-27 at 11 14 40 AM

~~That doesn't look great. Not sure why. Not sure if it's an issue with the dataset, model or visualization. That's the next step for me.~~

Here's an example train/validation loss curve with 400k parameters: </img>
